src/BloomExe/XmlHtmlConverter.cs

```diff
@@ -158,7 +147,16 @@ private static string RemoveFillerInEmptyElements(string contents)
 		}
 
 
+		public static string ConvertSelfClosingTags(string html, string innerHtml = "")
+		{
+			html = RemoveEmptySelfClosingTags(html);
+			return _selfClosingRegex.Replace(html, "<$1$2>" + innerHtml + "</$1>");
```

Comment:
Especially since the regex is not local, a comment saying "$1 is the tag, $2 is the attribute string" or similar would help.